subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/JavaInstallationProbe.java

```diff
@@ -93,9 +95,9 @@ private ProbeResult(InstallType installType, EnumMap<SysProp, String> metadata,
             this.error = error;
         }
 
-        public File getJavaHome() {
+        public Path getJavaHome() {
             assertOk();
-            return new File(metadata.get(SysProp.JAVA_HOME));
+            return Paths.get(metadata.get(SysProp.JAVA_HOME));
```

Comment:
Why is it called `SysProp.JAVA_HOME`?

Reply:
Because it is a reference to `java.home` system property?